### PR TITLE
Add more info for the ss14 game server status command

### PR DESCRIPTION
**Pull request by VasilisThePikachu**

### CODE REVIEW

File: gameserverstatus/gameserverstatus.py [hunk 1]
```diff
@@ -233,20 +233,23 @@ async def do_status_ss14(self, ctx: Messageable, cfgname: str, dat: Dict[str, st
...
+            countmax = json["soft_max_players"]
             name = json["name"]
+            round_id = json["round_id"]
+            gamemap = json["map"]
 
             if name:
                 embed.title = name
 
-            embed.add_field(name="Players Online", value=count)
+            embed.add_field(name="Players Online", value=f"{count}/{countmax}")
```
Comment (PJB3005) [inline gameserverstatus/gameserverstatus.py:247]: Please handle the case of `soft_max_players` not being reported.

File: gameserverstatus/gameserverstatus.py [hunk 2]
```diff
@@ -233,20 +233,23 @@ async def do_status_ss14(self, ctx: Messageable, cfgname: str, dat: Dict[str, st
...
         async with aiohttp.ClientSession() as session:
             async with session.get(addr + "/status") as resp:
-                json =await resp.json()
+                json = await resp.json()
 
             count = json["players"]
+            countmax = json["soft_max_players"]
             name = json["name"]
+            round_id = json["round_id"]
+            gamemap = json["map"]
```
Comment (PJB3005) [inline gameserverstatus/gameserverstatus.py:242]: Maybe make sure these are `.get()` so that if the server doesn't report them it's not an error.

**Review by PJB3005 — CHANGES_REQUESTED** [review]

**Comment by VasilisThePikachu** [comment]
I got no clue why GitHub won't embed those images and it's actually making me annoyed

**Comment by VasilisThePikachu** [comment]
okie it works on my fork as it turns out so it probably will work here too.

**Review by PJB3005 — APPROVED** [review]

**Pull request merged**